graphql/api/src/main/java/com/expediagroup/streamplatform/streamregistry/graphql/mutation/impl/ConsumerBindingMutationImpl.java

```diff
@@ -77,7 +80,12 @@ public Boolean delete(ConsumerBindingKeyInput key) {
   @Override
   public ConsumerBinding updateStatus(ConsumerBindingKeyInput key, StatusInput status) {
     ConsumerBinding consumerBinding = consumerBindingView.get(key.asConsumerBindingKey()).get();
-    return consumerBindingService.updateStatus(consumerBinding, status.asStatus()).get();
+
+    if (entityStatusEnabled) {
+      return consumerBindingService.updateStatus(consumerBinding, status.asStatus()).get();
+    } else {
+      return consumerBinding;
+    }
```

Comment:
All of the individual entity mutation implementation classes have this new Spring value toggle, set to true by default which is the current mode of operation.

When set to false an updateStatus mutation will NOT update the entity status via the entity service, it will just be a successful NOOP.